### PR TITLE
[V8 PORT] In @wdio/appium-service, kill entire Appium process tree

### PR DESCRIPTION
**Pull request by samuelfreiberg**
Porting to V8 branch.

[Original PR](https://github.com/webdriverio/webdriverio/pull/13151)

### CODE REVIEW

**Review by christian-bromann — COMMENTED** [review]
Something went wrong with the `pnpm-lock` file, mind taking a look?

File: pnpm-lock.yaml [hunk 1]
```diff
@@ -189,6 +189,1615 @@ importers:
         specifier: ^1.2.2
         version: 1.6.0(@types/node@22.2.0)(jsdom@22.1.0)
 
+  packages/eslint-plugin-wdio:
```
Comment (christian-bromann) [inline pnpm-lock.yaml:192]: This file should not be in here, we accidentally added it in on of our last PRs. Can you update your branch and call `npm install` again? Make sure it applies changes to `package-lock.json`

Comment (samuelfreiberg) [reply pnpm-lock.yaml]: @christian-bromann Updated

**Review by christian-bromann — APPROVED** [review]
LGTM 👍

**Comment by samuelfreiberg** [comment]
@christian-bromann How long does it usually take to create a new release? Is it automated?

**Comment by christian-bromann** [comment]
No, but I can get one out now.

**Comment by christian-bromann** [comment]
Released as `v8.40.3`.

**Pull request merged**